src/NuGet.Services.BasicSearch/Startup.cs

```diff
@@ -171,16 +171,16 @@ static Uri MakeRegistrationBaseAddress(string scheme, string registrationBaseAdd
             {
                 if (_searcherManager == null)
                 {
-                    await context.Response.WriteAsync("UNINITIALIZED");
                     context.Response.StatusCode = (int)HttpStatusCode.OK;
+                    await context.Response.WriteAsync("UNINITIALIZED");
```

Comment:
:+1: which is why in webapi a response is a single object that travels up the pipe. Such that if any error happens along the way of producing the result, you have a chance to replace the result with a 500 or another proper error that end user can see